controllers/results.js

```diff
@@ -6,7 +6,7 @@ var helpers = require('../lib/helpers');
 module.exports = {
 
   query: function query(req, res, next) {
-    if (req.query && (req.query.surname || req.query['system-number'])) {
+    if (req.query) {
```

Comment:
Is this the correct way to fix?

It seems to me that the real problem is the else clause itself. - I don't think it should redirect I think it should display an error page.